eng/common/scripts/Create-APIReview.ps1

```diff
@@ -83,6 +84,10 @@ else
 
 $FoundFailure = $False
 $pkgInfoPath = Join-Path -Path $ArtifactPath "PackageInfo"
+if ($ConfigFileDir)
+{
+    $pkgInfoPath = Join-Path -Path $ConfigFileDir "PackageInfo"
```

Comment:
I think you weren't understanding my other request. If ConfigFileDir is passed I'd expect it to be the exact directory where the config files exists and so we shouldn't append the PackageInfo path to those.